endpoints.json

```diff
@@ -23,5 +23,94 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves a single article",
+    "queries": [],
+    "exampleResponse": {
+      "article": [
+        {
+          "article_id": 1,
+          "title": "Living in the shadow of a great man",
+          "author": "weegembump",
+          "body": "Text from article",
+          "topic": "mitch",
+          "created_at": "2020-07-09T20:11:00.000Z",
+          "votes": 100,
+          "comment_count": 11
+        }
+      ]
+    }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "updates changes to the 'vote' value",
+    "queries": [{ "inc_votes": 10 }],
```

Comment:
`inc_votes` isn't a query, i'd add a row called `exampleRequest` and put it there. 

You will thank me when you do the front end and come back to reference this.